tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+
+Validate that location and filter mappings are complete
+    user waits until h3 is visible    Draft version tasks
+    user waits until parent contains element    testid:map-locations-task    link:Map locations
+    user waits until parent contains element    id:map-locations-task-status    text:Complete
+    user waits until parent contains element    testid:map-filters-task    link:Map filters
+    user waits until parent contains element    id:map-filters-task-status    text:Complete
+
+Confirm finalization of this API data set version
+    user clicks button    Finalise this data set version
+    user waits for caches to expire
+    user waits until h2 is visible    Mappings finalised
+    user waits until page contains    Draft API data set version is ready to be published
+
+User navigates to 'changelog and guidance notes' page and update relevant details in it
+    user clicks link by index    View changelog and guidance notes    1
+    user waits until page contains    API data set changelog
+
+    user enters text into element    css:textarea[id="guidanceNotesForm-notes"]
+    ...    Content for the public guidance notes
+    user clicks button    Save public guidance notes
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+User clicks on 'View preview token log' link inside the 'Draft version details' section
+    user clicks link by index    View changelog and guidance notes    2
+
+Validate the contents in the 'API dataset changelog' page.
+    user waits until page contains    API data set changelog
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+Add headline text block to Content page for the second release
+    user navigates to content page    ${PUBLICATION_NAME}
+    user adds headlines text block
+    user adds content to headlines text block    Headline text block text
+
+Approve second release
+    user clicks link    Sign off
+    user approves release for immediate publication
+
+Verify newly published release is on Find Statistics page
+    user checks publication is on find statistics page    ${PUBLICATION_NAME}
+
+User navigates to data catalogue page
+    user navigates to data catalogue page on public frontend
+
+Search with 2nd API dataset
+    user clicks element    id:searchForm-search
+    user presses keys    ${PUBLICATION_NAME}
+    user clicks radio    API data sets only
+
+    user waits until page finishes loading
+    user clicks radio    Newest
+
+    user checks summary list contains    Status    This is the latest data
+    ...    parent=testid:data-set-file-summary-${SUBJECT_2_NAME}
+    user checks summary list contains    Status    Available by API
+    ...    parent=testid:data-set-file-summary-${SUBJECT_2_NAME}
+    user checks page contains link    ${SUBJECT_2_NAME}
+
+User clicks on 2nd API dataset link
```

Comment:
As above - should this be `User clicks on API data set link`?

Reply:
See prev comment.  Now referred to as "new data set version"